cms/djangoapps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -366,4 +366,4 @@ def test_rerun_duplicate_run(self):
         }
         response = self.client.post(url, data, format='json')
         assert response.status_code == 400
-        assert response.data == {'run': [u'Course run {key} already exists'.format(key=course_run.id)]}
+        assert response.data == {'non_field_errors': [u'Course run {key} already exists'.format(key=course_run.id)]}
```

Comment:
Please do not break compatibility. It's fine to add new parameters and errors. Please don't move the old one.

Reply:
Ah, my bad! That became the default key value when we switched from `validate_run` to `validate`. Will fix up